### PR TITLE
For review PR

### PR DESCRIPTION
**Pull request by bryancross**
No actual changes, just the code itself for review

### CODE REVIEW

**Comment by bryancross** [comment]
So now there's changes.  Fixes #7

File: script/bootstrap.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
-#npm install
+npm install
 echo "creating log directory"
 mkdir -p log
 echo "Creating jobs directory"
 mkdir -p jobs
 cp config/job-template-example.json config/job-temlplate.json
```
Comment (tcbyrd) [inline script/bootstrap.sh]: s/temlplate/template

**Comment by tcbyrd** [comment]
Couldn't leave a line comment, but there's an extra `d` at the end of the config template that's causing an error when running `script/server`

https://github.com/bryancross/key2pdf/blob/master/config/job-template-example.json#L32

```
SyntaxError: /Users/tcbyrd/key2pdf/config/job-template.json: Unexpected token d in JSON at position 649
```

**Comment by tcbyrd** [comment]
@bryancross I just pulled down the changes after https://github.com/bryancross/key2pdf/pull/6/commits/9e6c6c14a0ee15f89c81dc1cb8ae5e0feff1d8ed and everything seems to be working on my end! 👍

**Comment by bryancross** [comment]
Awesome!  The more I go through this and the more I learn the more I realize what a mess this is.  But, it does seem to have the 'working' feature, mostly, which is important.  

I'ma merge this and start another branch to actually add the cataloging feature, among other things.  Thanks for your help!

**Pull request merged**